objects/screen.c

```diff
@@ -1655,6 +1658,60 @@ luaA_screen_get_name(lua_State *L, screen_t *s)
     return 1;
 }
 
+/** Get the content of the active screen as a cairo surface
```

Comment:
ldoc sometime is weird when it comes to find the limit of the description. It's better to always add a final `.` at the end of the summary:

```suggestion
/** Get the content of the active screen as a cairo surface.
```

Reply:
As I am not a project member, I am going to believe you on all these sorts of things. I tried to basically mimic what was in the source files, but I have only been looking at them for a few days now.